Domain/ProductManagement/Inventory.cs

```diff
@@ -0,0 +1,211 @@
+﻿using LenaInventoryManagementSystem.Domain.ProductManagement;
+
+namespace LenaInventoryManagementSystem.Domain
+{
+    public class Inventory
+    {
+        private static List<Product> _products = new();
+        private static InventoryUI _inventoryUI=new();
+        internal static void MainMenu()
+        {
+            _inventoryUI.DisplayMainMenu();
+            var userSelection=_inventoryUI.UserSelection();
+
+            switch (userSelection)
+            {
+                 case "1":
+                    AddANewProduct();
+                    break;
+                case "2":
+                    ViewAllProducts();
+                    break;
+                case "3":
+                    UpdateAProduct();
+                    break;
+                case "4":
+                    DeleteAProduct();
+                    break;
+                case "5":
+                    ViewAProduct();
+                    break;
+                default:
+                    _inventoryUI.PrintMessage("Invalid selection. Please try again.");
+                    break;
+            }
+        }
```

Comment:
We should have new lines between functions and constructors. lets do this change to all methods in this PR.